definitions/private-law/json/AuthorisationCaseField/RespondentSolicitor/Miam/AuthorisationCaseField.json

```diff
@@ -146,5 +146,92 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "LiveFrom": "02/12/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "hasRespondentAttendedMiam",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[C100RESPONDENTSOLICITOR1]",
+          "[C100RESPONDENTSOLICITOR2]",
+          "[C100RESPONDENTSOLICITOR3]",
+          "[C100RESPONDENTSOLICITOR4]",
+          "[C100RESPONDENTSOLICITOR5]"
```

Comment:
We need to update system user CRU access for all

Reply:
Done.